samples/beep_test.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Was this file supposed to go in? If so, should we rename it in the same pattern as the other JSONs? Or perhaps we should actually also rename the other JSON sampes to give more descriptive names.